### PR TITLE
Project: Embedding videos and images

### PR DESCRIPTION
**Pull request by evansinho**
#### In this milestone we :
- Embedded the videos.
- Added Navigation bar with a Search button
- Embedded the images on the recommended video section.
- Added footer.
- Validated the HTML and CSS files.

### CODE REVIEW

**Comment by SofiBretz** [comment]
### Changes required ♻️
### Hello team 🌟

Great work on this project, but you need to fix the issues below before moving on ⬇️

- [ ] The subscribe button has to be at the right of that section, consider following the original design.

- [ ] Add, new, Share, and More button has to be at the right of that section, consider following the original design.

- [ ] views and date have to be at the left side of the section, consider following the original design.

- [ ] on the navbar the 'sign in' icon and the 3 other icons have to be at the very right side as the original page.

- [ ] the search bar has to be center, it seems to be center but when you fix the navbar icons you have to center the search bar more. 

- [ ] Make sure that basic CSS bad practices are not used ([article](https://speckyboy.com/good-bad-css-practices/) for support ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

no repeating of the same rule
no ID selectors for multiple elements
no !import overusing
no inefficient selectors
no inline styling

_**After making the required changes please submit a new code review request**_
Good luck! 🍀 You are almost there! 💪

SofiBretz.

**Comment by uryelah** [comment]
Hello **Evanson Igiri** & **Taiwo Coker**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all, welcome to Microverse and good job working on the changes of your first project🌟**

Your project looks great design and markup wise, you are just missing one requirement. You got this! 💪

---------    

## Required Changes 🏗 

- [ ] What the last TSE meant about the CSS best practices was probably about the overuse of id selectors in your style sheet.  Please make sure you use class selectors instead. 
You can read more about [why using id selectors is not the best idea here.](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4)

- [ ] [OPTIONAL] Add a 'cursor: pointer' property to your buttons and icons on the page, this will make the page more user friendly.

----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

File: style.css [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+  padding-bottom: 20px;
+}
+
+p {
+  margin: 10px 0;
+}
+
+/****** Navigation Bar *****/
+
+#navbar {
```
Comment (uryelah) [inline style.css]: Use `.navbar` instead.

**Comment by uryelah** [comment]
Good job with the fix guys!

------
## Your project is approved! 🎊 🥅 💛 
### 
**You may close this Pull Request and merge the changes**

-------

**Keep being an awesome dev and good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**